src/app/main/services/budgets.service.ts

```diff
@@ -140,6 +140,18 @@ export class BudgetsService {
     return refSnapshot;
   }
 
+   updateBudgetStatus(
+        id: string,
+        status: any
+        ): Observable<firebase.default.firestore.WriteBatch>{
+          const batch = this.afs.firestore.batch();
+          const docRef: DocumentReference = this.afs.firestore.doc(`/db/ferreyros/budgets/${ id }`);
+
+          batch.update(docRef, status);
+
+          return of(batch);
+  }
```

Comment:
@Msartor-meraki-s 
Aquí debes actualizar el método para recibir el `id: string` y `status: string`. Líneas más abajo debes construir el objeto que actualizaras.

```suggestion
   updateBudgetStatus(
        id: string,
        status: string
        ): Observable<firebase.default.firestore.WriteBatch>{
          const batch = this.afs.firestore.batch();
          const docRef: DocumentReference = this.afs.firestore.doc(`/db/ferreyros/budgets/${ id }`);
          const data = {
           statusPresupuesto: status
          };
          batch.update(docRef, data);
          return of(batch);
  }
```